lambdas/handlers/authoriser_handler.py

```diff
@@ -63,27 +61,54 @@ def lambda_handler(event, context):
     principal_id = ""
     _, _, _, region, aws_account_id, api_gateway_arn = event["methodArn"].split(":")
     api_id, stage, _http_verb, _resource_name = api_gateway_arn.split("/")
-
+    user_role = decoded["repository_role"]
```

Comment:
Is there a reason this is outside the try-except? code there be a keyError here?

Reply:
We sure can, updated in next push.